src/ml_board/frontend/Dashboard/src/worker_socket/MainThreadCallbacks.ts

```diff
@@ -0,0 +1,66 @@
+import { DataFromSocket, DataToRedux } from "./DataTypes";
+import { MLGYM_EVENT } from "./EventsTypes";
+import handleEvaluationResultData, { evalResultCustomData, EvaluationResultPayload } from "./event_handlers/evaluationResultDataHandler";
+import handleExperimentStatusData from "./event_handlers/ExperimentStatusHandler";
+import handleJobStatusData from "./event_handlers/JobStatusHandler";
+
+// Note: I don't like having anything here other than the callbacks but it is what it is :')
+// ========================= variables ============================//
+
+// TODO: temporary until the mystery why this is even used be solved
+const initialStateForGraphs: evalResultCustomData = {
+    grid_search_id: null,
+    experiments: {},
+    colors_mapped_to_exp_id: {}
+};
+
+
+// ========================= Callbacks to update the MainThread ============================//
+// ========================= MTCb == MainThreadCallback ====================================//
+
+export const updateMTCb = (parsedSocketData: DataFromSocket) => {
```

Comment:
rename to `updateMainThreadCallback` MT is ambiguous

Reply:
the best code, is code that is self-explanatory and does not need almost any comments :-)
Imagine someone using this function. This person would have to through the code and check the comment to understand what it does
.